lib/components/mobile/results-screen.js

```diff
@@ -175,18 +168,14 @@ const mapStateToProps = (state, ownProps) => {
     error: getActiveError(state),
     query: state.otp.currentQuery,
     realtimeEffects,
-    resultCount:
-      response
-        ? activeSearch.query.routingType === 'ITINERARY'
-          ? itineraries.length
-          : response.otp.profile.length
-        : null,
-    useRealtime
+    resultCount: response
+      ? activeSearch.query.routingType === 'ITINERARY'
+        ? itineraries.length
+        : response.otp.profile.length
+      : null
   }
 }
 
-const mapDispatchToProps = {
-  setUseRealtimeResponse: narrativeActions.setUseRealtimeResponse
-}
+const mapDispatchToProps = {}
```

Comment:
Delete this var and the 2nd arg passed to `connect`.

Reply:
Good call thanks! Cleaned up in 23ef058a46835bd8ce5710176ca1671ffcf7a586